.github/workflows/fork.yaml

```diff
@@ -0,0 +1,77 @@
+name: bump firecracker
+
+on:
+  workflow_dispatch: {}
+  schedule:
+  - cron: 0 0 * * 1
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        ssh-key: ${{ secrets.WEAVEWORKSBOT_TOKEN }}
+        fetch-depth: 0
+    - name: Bump firecracker
+      run: |
+        new_version=$(./hack/scripts/bump_firecracker.sh)
+        if [[ "$new_version" == "" ]]; then
+          echo "No new Firecracker release. Nothing to do. (this is not an error, no worries.)"
+          exit 1
+        fi
+
+        echo 'NEW_VERSION<<EOF' >> $GITHUB_ENV
+        echo "${new_version}" >> $GITHUB_ENV
+        echo 'EOF' >> $GITHUB_ENV
+    - name: Create PR
+      if: ${{ success() }}
```

Comment:
Instead of `if: ${{ success() }}` i'm wondering if you change the expression so that this is only executed if there is a new version, so something like  `if: ${{ env.NEW_VEWRSIOn != "" }}`  and then in the **Bump firecracker** step we do do away with the `exit 1`

Reply:
yeh that sounds better 👍